includes/Story_Post_Type.php

```diff
@@ -530,7 +547,9 @@ public function get_editor_settings() {
 				'allowedImageMimeTypes' => $this->get_allowed_image_mime_types(),
 				'allowedMimeTypes'      => $this->get_allowed_mime_types(),
 				'postType'              => self::POST_TYPE_SLUG,
+				'userId'                => $user_id,
```

Comment:
As per my other comment, I don't think we actually need this.

But if we do, we should just grab it from `CurrentUserProvider`

Reply:
Again, think we need to move the compontent out of it's current location to where it was before, meaning we do still need this.